### ISSUE TITLE
[Feature request] Shortcuts supporn

### ISSUE THREAD

**Issue by sunnaryt** [F4uzan/HgLauncher#45]
App shortcuts do not yet work, this is inconvenient for people who like the added sandboxing of web apps or for people who use apps like Shelter which relies on app shortcuts to launch applications installed on the Shelter profile conveniently.

**Comment by macdonaldster** [comment]
Hi, can you please explain the issue more?

I can try to reproduce this but want to make sure I am doing it correctly. 

Is this the app you are talking about? https://f-droid.org/en/packages/net.typeblog.shelter/

**Comment by F4uzan** [comment]
I think I understand the issue. Shelter uses shortcuts to handle launching of its sheltered apps, and without this, it becomes cumbersome to launch apps there.

App shortcut support should be imminent. I plan to finish up for v1.4.2.2, then work for v1.4.3, which should include app shortcut support (hopefully).

**Comment by sunnaryt** [comment]
Yes, that's the app and purpose for my concern, as well as (what I believe to be the same function) the function of various web browsers to create web shortcuts as if they are apps (the reason this is nice is you can use Uber like this, it opens without browser borders, which looks pretty seamless and has added sandboxing).

Anyway, thank you very much, I look forward to installing this on my daily use phone after that's implemented.
